display_list/display_list_matrix_clip_tracker.cc

```diff
@@ -80,12 +81,15 @@ class Data3x3 : public DisplayListMatrixClipTracker::Data {
     FML_CHECK(false) << "SkM44 was set without upgrading Data";
   }
   void setIdentity() override { matrix_.setIdentity(); }
+  bool mapRect(const SkRect& rect, SkRect* mapped) const override {
+    return matrix_.mapRect(mapped, rect);
+  }
+  virtual bool canBeInverted() const override {
```

Comment:
nit - virtual not needed here (I was wondering why this line was longer than the Data4x4 version)

Reply:
Ahh, I see, good catch, thanks!